### PR TITLE
Read blasts information from a gSheet

### PR DESCRIPTION
**Pull request by nerdlet**
##### What does this PR do?
=>Add ` tabletop ` to pull data from google sheet and 
 display on the graph: 
Google sheet: https://docs.google.com/spreadsheets/d/1DTK8hyI1sGooGOU-fQh-amYGc7iJk4Bc6QlARfXuE8U/edit?usp=sharing
##### Screenshots?

image

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
Are we using the gSheet __just__ to get the number of points @nerdlet ? Seems a lot of work for very little payoff.

What about `Last Uploaded` date? Can't it come from the gSheet as well?

**Comment by nerdlet** [comment]
@kilemensi Sure...last uploaded date right now

File: src/components/Blasts/BlastVisual.js [hunk 1]
```diff
@@ -232,7 +232,6 @@ class BlastVisual extends React.Component {
     const { blastData } = this.state;
 
     const lastArray = blastData[blastData.length - 1];
-    console.log(lastArray);
     const date = lastArray.Date;
     const d = date.split(' ')[0];
     console.log(d);
```
Comment (karimkawambwa) [inline src/components/Blasts/BlastVisual.js]: this @nerdlet

File: src/components/Blasts/BlastVisual.js [hunk 2]
```diff
@@ -201,61 +202,92 @@ const styles = theme => ({
...
-    const date = new Date();
-    const today = date.toLocaleTimeString();
-    return today;
+function CurrentDate() {
+  const date = new Date();
+  const today = date.toLocaleTimeString([], {
+    hour: '2-digit',
+    minute: '2-digit'
+  });
+  return today;
```
Comment (kilemensi) [inline src/components/Blasts/BlastVisual.js]:
```javascript
const currentDate = () => new Date().toTimeString().substring(0,5)
```

File: src/components/Blasts/BlastVisual.js [hunk 3]
```diff
@@ -201,61 +202,92 @@ const styles = theme => ({
...
+    });
+  }
+
+  render() {
+    const { classes } = this.props;
+    const { blastData } = this.state;
+
+    const lastArray = blastData[blastData.length - 1];
+    const date = lastArray.Date;
+    const d = date.split(' ')[0];
```
Comment (kilemensi) [inline src/components/Blasts/BlastVisual.js]:
```javascript
...
    const lastBlast = blastData[blastData.length - 1];
    const datetime = lastBlast.Date;
    const date = datetime.split(' ')[0];
....
```

May need to check _if_ we actually have `datetime` and default to _some_ value *if* we don't.

File: src/components/Blasts/BlastVisual.js [hunk 4]
```diff
@@ -201,61 +202,92 @@ const styles = theme => ({
...
-              <FontAwesomeIcon icon={faFacebookF} size="2x" />
-            </FacebookShareButton>
+
+  componentDidMount() {
+    Tabletop.init({
+      key: '1DTK8hyI1sGooGOU-fQh-amYGc7iJk4Bc6QlARfXuE8U',
+      callback: googleData => {
+        this.setState({ blastData: googleData });
+      },
+      simpleSheet: true
```
Comment (kilemensi) [inline src/components/Blasts/BlastVisual.js]: What does `simpleSheet` do?

**Review by kilemensi — APPROVED** [review]
:+1: @nerdlet... address the comments and merge.

**Comment by nerdlet** [comment]
@kilemensi Done

**Pull request merged**